README.md

```
@@ -64,6 +64,15 @@ named `public.target_table`.
 
 6. Run! `$ uv run subset`
 
+Or, from PyPI:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)